### PR TITLE
feat: added stop function to library

### PR DESCRIPTION
**Pull request by dan-mckay**
stop feature also added to playground app

### CODE REVIEW

**Comment by dan-mckay** [comment]
I just created a new branch from the origin and squashed it all into a single commit - keeping it tidy!

File: playground/app.es6 [hunk 1]
```diff
@@ -11,6 +11,7 @@ import React from 'react';
 import thunkMiddleware from 'redux-thunk';
 
 const loggerMiddleware = createLogger({
+  predicate: (getState, action) => action.type !== tracks.actions.TICK,
```
Comment (dariocravero) [inline playground/app.es6:14]: good call :)

File: playground/nodes.es6 [hunk 2]
```diff
@@ -45,6 +45,11 @@ export function play(id, tick) {
   }
 }
 
+export function stop(id) {
+  pause(id);
```
Comment (dariocravero) [inline playground/nodes.es6:49]: I think it's cleaner this way. Thoughts?

**Comment by dariocravero** [comment]
Thanks @dmcaodha released as `v1.3.1`!

File: playground/nodes.es6 (hunk 2 above)
Comment (dan-mckay) [inline playground/nodes.es6:49]: makes sense alright, I'm good with that

**Pull request merged**